ChopChop/Storage/FirebaseCloudStorage.swift

```diff
@@ -0,0 +1,45 @@
+import FirebaseStorage
+import UIKit
+import Combine
+
+struct FirebaseCloudStorage {
+    let storageRef = Storage.storage().reference()
+    static let imageMaxSize: Int64 = 1_000 * 1_024 * 1_024
+
+    func uploadImage(image: UIImage, name: String) {
+        let uploadRef = storageRef.child("images/\(name).png")
+        guard let uploadData = image.pngData() else {
+            return
+        }
+        let metaData = StorageMetadata()
+        metaData.contentType = "image/png"
+        uploadRef.putData(uploadData, metadata: metaData) { _, error in
+            if error != nil {
+                debugPrint("error")
```

Comment:
In what situation would there be an error? Does it warrant a better way to handle the error here?

Reply:
Prob some firebase connection error or may be the quota exceeded. Not sure how to handle error here cos the closure does not allow throwing of errors.